### PR TITLE
Safe delete for cloud volume

### PR DESCRIPTION
**Pull request by gregoryb-at-ibm**
This PR adds the raw safe deletion to the cloud-volume of this provider.

Part of https://github.com/ManageIQ/manageiq/issues/21055

### CODE REVIEW

**Comment by llivne** [comment]
LGTM. I will megre after all of the linked tickets will be merged and Travis will pass

File: app/models/manageiq/providers/autosde/storage_manager/cloud_volume.rb
Comment (agrare) [inline app/models/manageiq/providers/autosde/storage_manager/cloud_volume.rb:57]: @gregoryb-at-ibm can you remove this since we don't need validate_safe_delete_volume anymore?

**Review by agrare — APPROVED** [review]
Going to merge this to unblock the API change and this extra method isn't hurting anyone

https://travis-ci.com/github/ManageIQ/manageiq-api/jobs/484246230#L2040-L2057

**Comment by agrare** [comment]
(merging red since cross-repo tests were green)

**Pull request merged**
